sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/jdbc/JDBCRDD.scala

```diff
@@ -72,13 +72,25 @@ object JDBCRDD extends Logging {
           JdbcUtils.getSchema(rs, dialect, alwaysNullable = true,
             isTimestampNTZ = options.preferTimestampNTZ)
         } finally {
-          rs.close()
```

Comment:
Can we use ` AutoCloseable` syntax instead? That will show the original exception as well.

Reply:
+1 for the above comment.

Reply:
Cool! Thank you @HyukjinKwon @dongjoon-hyun . PR updated.